packages/ie11-polyfills/index.js

```diff
@@ -0,0 +1 @@
+export * from 'ie11-custom-properties';
```

Comment:
If we're going to use ES6 export syntax like this, the package will need to use TS to transpile (not everything can handle ES6 exports). Also, it would be better to just do `import 'ie11-custom-properties'` since that package relies on side effects rather than exporting anything.

The other option would be to keep it as JS and change this line to `require('ie11-custom-properties')`.

I can see arguments for either keeping as JS (mainly that there's no build cost) or converting to TS (which would probably be needed if we add more polyfills later). Unless someone else has strong feelings I think we should stick with JS for now.

Reply:
Will stick with JS for now